Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -746,7 +746,6 @@ extension NSManagedObjectContext: MessageDatabaseSession {
         }
 
         message.reactionScores[type.rawValue] = max(reactionScore - Int(reaction.score), 0)
-        message.reactionScores[type.rawValue] = reactionScore
```

Comment:
this is no code smell, this is a bug... i wonder what were the implications & which line is actually intended

Reply:
The max comparison is the required LOC, but yes I couldn't figure out if any bad behaviour with either